src/commands/tileindex-validate/tileindex.validate.ts

```diff
@@ -478,14 +518,10 @@ export function validateTiffAlignment(tiff: TiffLocation, allowedError = 0.015):
 }
 
 export function getTileName(x: number, y: number, gridSize: GridSize): string {
-  const offsetX = Math.round(Math.floor((x - MapSheet.origin.x) / MapSheet.width));
-  const offsetY = Math.round(Math.floor((MapSheet.origin.y - y) / MapSheet.height));
-
-  // Build name
-  const letters = Object.keys(SheetRanges)[offsetY];
-  const sheetCode = `${letters}${`${offsetX}`.padStart(2, '0')}`;
-  // TODO: re-enable this check when validation logic
-  // if (!MapSheet.isKnown(sheetCode)) throw new Error('Map sheet outside known range: ' + sheetCode);
+  const sheetCode = MapSheet.sheetCode(x, y);
+  if (!MapSheet.isKnown(sheetCode)) {
+    logger.info(`Map sheet (${sheetCode}) at coordinates (${x}, ${y}) is outside the known range.`);
```

Comment:
these should be extracted into log parameters, `logger.info({ sheetCode}, '...')`

Reply:
fixed